pom.xml

```diff
@@ -0,0 +1,236 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+   <description>OpenMetadata Spark Agent</description>
+   <modelVersion>4.0.0</modelVersion>
+   <groupId>org.open-metadata</groupId>
+   <artifactId>openmetadata-spark-agent</artifactId>
+   <packaging>jar</packaging>
+   <version>1.0-SNAPSHOT</version>
+   <name>${project.groupId}:${project.artifactId}</name>
+   <url>https://github.com/open-metadata/OpenMetadata</url>
+   <licenses>
+      <license>
+         <name>The Apache License, Version 2.0</name>
+         <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+      </license>
+   </licenses>
+   <developers>
+      <developer>
+         <name>OpenMetadata</name>
+         <email>openmetadata@getcollate.io</email>
+         <organization>Get Collate</organization>
+         <organizationUrl>https://getcollate.io/</organizationUrl>
+      </developer>
+  </developers>
+   <scm>
+    <connection>scm:git:git@github.com:Open-Metadata/OpenMetadata</connection>
+    <developerConnection>scm:git:git@github.com:Open-Metadata/OpenMetadata</developerConnection>
+    <url>scm:git:git@github.com:Open-Metadata/OpenMetadata</url>
+  </scm>
+   <properties>
+      <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+      <maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
should this java 17? not sure what spark's java version it supports

Reply:
17 is not supported but 11 is, so have changed it to 11